public/locales/en/administration.json

```diff
@@ -84,10 +86,12 @@
         "shouldHaveAccess": "If you think you should have access to this page, please contact a site administrator or a member of the South Gate CEHAT."
     },
     "manageAccount":{
-        "methodFetchError": "Error occurred when fetching user sign in methods",
         "connectedToGoogle": "Account connected to Google",
         "heading": "Manage Your Account",
-        "manageSignInMethodsHeader": "Manage Sign In Methods"
+        "manageSignInMethodsHeader": "Manage Sign In Methods",
+        "sendEmailVerificationEmail": "Send Email Verification Email",
```

Comment:
Is supposed to just be "Send Verification Email" (it seems like that's what it is in Spanish). Also change the key to "sendVerificationEmail"